render/src/rendered.rs

```diff
@@ -106,6 +106,11 @@ impl XSynthRender {
             );
             self.channel_group
                 .read_samples(&mut self.render_elements.output_vec);
+
+            if let Some(limiter) = &mut self.limiter {
```

Comment:
This was just an oversight on my part right?

Reply:
I don't remember who wrote this, but yeah I remember having this exact issue in Forte sometime too but it was never applied here I guess